src/test/java/org/jabref/gui/util/ThemeTest.java

```diff
@@ -0,0 +1,258 @@
+package org.jabref.gui.util;
+
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.StandardOpenOption;
+import java.util.Optional;
+
+import javafx.collections.FXCollections;
+import javafx.scene.Scene;
+
+import org.jabref.preferences.AppearancePreferences;
+import org.jabref.preferences.PreferencesService;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.condition.EnabledOnOs;
+import org.junit.jupiter.api.condition.OS;
+import org.junit.jupiter.api.io.TempDir;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+import static org.junit.jupiter.api.Assertions.fail;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.when;
+
+public class ThemeTest {
+
+    private Path tempFolder;
+
+    private PreferencesService preferencesMock;
+
+    @BeforeEach
+    void setUp(@TempDir Path tempFolder) {
+        this.tempFolder = tempFolder;
+        this.preferencesMock = mock(PreferencesService.class);
+        AppearancePreferences appearancePreferences = mock(AppearancePreferences.class);
+        when(this.preferencesMock.getAppearancePreferences()).thenReturn(appearancePreferences);
+        when(appearancePreferences.shouldOverrideDefaultFontSize()).thenReturn(false);
+    }
+
+    @Test
+    public void lightThemeUsedWhenPathIsBlank() {
+        Theme blankTheme = new Theme("", preferencesMock);
+        assertEquals(Theme.Type.LIGHT, blankTheme.getType());
+        blankTheme.additionalStylesheet().ifPresent(location -> fail(
```

Comment:
`assertEquals(Optional.empty(), blankTheme.additionalStylesheet())` should work as well.

Reply:
Updated to use this assertion